integrations/ultralytics-yolov5/README.md

```diff
@@ -1,51 +1,30 @@
-<!--
-Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing,
-software distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
--->
-<span style="color:red">This pathway is deprecated. Please refer to the instructions [here](https://github.com/neuralmagic/sparseml/blob/main/src/sparseml/yolov5) on how to run YOLOv5 + SparseML with `sparseml.yolov5` commands.</span>
-# SparseML Ultralytics YOLOv5 Integration
-
+# SparseML YOLOv5 Integration
 This directory combines the SparseML recipe-driven approach with the 
 [ultralytics/yolov5](https://github.com/ultralytics/yolov5) repository.
-By integrating the robust training flows in the `yolov5` repository with the SparseML code base,
-we enable model sparsification techniques on the popular [YOLOv5 architecture](https://github.com/ultralytics/yolov5/issues/280)
+By integrating the robust training flows in the `yolov5` repository with the SparseML code base, we enable model sparsification techniques on the popular [YOLOv5 architecture](https://github.com/ultralytics/yolov5/issues/280)
+as well as the updated [YOLOV5-P6 architecture](https://github.com/ultralytics/yolov5/releases/tag/v5.0),
 creating smaller and faster deployable versions.
 The techniques include, but are not limited to:
 - Pruning
 - Quantization
-- Pruning + Quantization
 - Sparse Transfer Learning
 
-## Highlights
+After training, the model can be deployed with Neural Magic's DeepSparse Engine. The engine enables inference with GPU-class performance directly on your CPU.
 
-- Example: [DeepSparse YOLOv5 Inference Example](https://github.com/neuralmagic/deepsparse/tree/main/examples/ultralytics-yolo)
+This integration enables spinning up one of the following end-to-end functionalities:
+- **Sparsification of YOLOv5 Models** - easily sparsify any of the YOLOV5 and YOLOV5-P6 models, from YOLOv5n to YOLOv5x models. 
```

Comment:
The wording here is a bit confusing for me. I know that n in "YOLOv5n" refers to the model size, but may be a bit confusing to a new user.